receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -223,6 +223,14 @@ attributes:
     description: "the name of Kubernetes Node condition. Example: Ready, Memory, PID, DiskPressure"
     type: string
 
+  k8s.container.status.state:
```

Comment:
I know we haven't done this so far, but should we add links here to the respective SemConv definitions for reference?